src/Cloud/weglide/UploadFlight.cpp

```diff
@@ -31,25 +31,25 @@
 #include "Formatter/TimeFormatter.hpp"
 #include "json/ParserOutputStream.hxx"
 #include "system/ConvertPathName.hpp"
-#include "system/Path.hpp"
+#include "LocalPath.hpp"
 #include "util/StaticString.hxx"
 
 #include <cinttypes>
 
 namespace WeGlide {
 
 static CurlMime
-MakeUploadFlightMime(CURL *easy, const WeGlideSettings &settings,
+MakeUploadFlightMime(CURL *easy, const WeGlide::Pilot &pilot,
                      uint_least32_t glider_type,
                      Path igc_path)
 {
   CurlMime mime{easy};
-  mime.Add("file").Filename("igc_file").FileData(NarrowPathName{igc_path});
+  mime.Add("file").Filename("igc_file").FileData(NarrowPathName{LocalPath(igc_path)});
```

Comment:
Layering violation: this low-level library is not supposed to call into the glue library `LocalPath.hpp`.
This change is not documented in the commit message, and seems to be completely unrelated to the rest of the commit.

Reply:
Yes, this was wrong -> removed